test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -333,15 +340,56 @@ public void testCrNocr() {
 	/* test options in runnable jar files */
 	
 	public void testJarArguments() {
-		String cmdlineArg = "-DcmdlineArg";
-		String testJar = System.getProperty("org.openj9.test.vmArguments.testJar");
-		URL jarUrl = ClassLoader.getSystemClassLoader().getResource("vmargs.jar");
-		assertNotNull("vmargs.jar not found", jarUrl);
-		testJar = jarUrl.getFile();
-		if (isWindows() && testJar.startsWith("/")) {
-			testJar = testJar.substring(1); /* annoying habit of getResource() to put a / at the front of the path */
+		URL jarUrl = ClassLoader.getSystemClassLoader().getResource(VMARGS_JAR);
+		assertNotNull(VMARGS_JAR+" not found", jarUrl);
+		Path jarPath = trimLeadingSlash(jarUrl);		
+		ProcessBuilder pb = makeProcessBuilder(null, new String[] {cmdlineArg, "-jar", jarPath.toString()}, null);
+		Map<String, String> env = pb.environment();
+		String javaToolOptionsArg = "-DjavaToolOptionsArg";
+		env.put(JAVA_TOOL_OPTIONS, javaToolOptionsArg );
+		String ibmJavaOptionsArg = "-DibmJavaOptionsArg";
+		env.put(IBM_JAVA_OPTIONS, ibmJavaOptionsArg );
+		ArrayList<String> actualArguments = runAndGetArgumentList(pb);
+		String fooBar = "-Dfoo=bar";
+		String longProp = "-Da.long.system.property=this is a long system property value to demonstrate long JVM arguments in the manifest file";
+		HashMap<String, Integer> posns = checkArguments(actualArguments, new String[] {javaToolOptionsArg, ibmJavaOptionsArg, cmdlineArg, fooBar,
```

Comment:
`posns` please void truncated names like this unless they are commonly used / referenced.  It's better for the reader of the code if this is called `positions` as they don't have to guess at the meaning of the name.

The use of `env` earlier is fine though as its a very common abbreviation.